doc/release-notes/0.46/0.46.md

```diff
@@ -0,0 +1,156 @@
+<td valign="top">Support is added for the <tt>com.sun.management.ThreadMXBean.getTotalThreadAllocatedBytes()</tt> API.</td>
+<td valign="top">All versions</td>
+<td valign="top">The OpenJ9 VM implementation supports measurement of the total memory allocation for all threads (<tt>com.sun.management.ThreadMXBean.getTotalThreadAllocatedBytes()</tt> API). The <tt>getTotalThreadAllocatedBytes()</tt> method now returns the total thread allocated bytes instead of <tt>-1</tt>.</td>
+</tr>
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/19585">#19585</a>, <a href="https://github.com/eclipse-openj9/openj9/pull/19619">#19619</a></td>
+<td valign="top">The JITServer AOT caching feature is enabled by default at the JITServer server.</td>
+<td valign="top">All versions (Linux)</td>
+<td valign="top"><tt>-XX:+JITServerUseAOTCache</tt> is the default setting at the JITServer server now. That means that you don't have to specify the <tt>-XX:+JITServerUseAOTCache</tt> option at the server to enable the JITServer AOT caching feature.
+
+Although this option is by default enabled at the server, it is still disabled for the JITServer clients. The clients that want to use the JITServer AOT caching, must still specify the <tt>-XX:+JITServerUseAOTCache</tt> option on the command line. Also, now the clients don't have to enable the shared classes cache feature to use the <tt>-XX:+JITServerUseAOTCache</tt> option.
+</td>
+</tr>
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/19554">#19554</a></td>
+<td valign="top">A new option <tt>-XX:[+|-]EnableExtendedHCR</tt> is added.</td>
+<td valign="top">All versions</td>
+<td valign="top">By default, the extended Hot Code Replace (HCR) capability in the VM is disabled for all OpenJDK versions. You can enable or disable the HCR capability by using the <tt>-XX:[+|-]EnableExtendedHCR</tt> option.
+
+The extended HCR feature is deprecated in this release and will be removed in a future release. From OpenJDK 25 onwards, extended HCR will not be supported. Following that, the extended HCR support will be removed from other earlier OpenJDK versions also.
+</td>
+</tr>
+</tbody>
+</table>
+
+## Known issues
+
+The v0.46.0 release contains the following known issues and limitations:
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1">
+<thead align="left">
+<tr>
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+<th valign="bottom">Workaround</th>
+</tr>
+</thead>
+
+<tbody>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/19833">#19833</a></td>
+<td valign="top">The class comparison might not detect the removal of method access modifiers. For example, a change of a method from public to package-private.</td>
```

Comment:
This needs to be more specific, i.e. "Class comparison for the shared cache might not ..."